src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowDelete.cs

```diff
@@ -128,6 +128,9 @@ private bool CheckForDuplicateDeleteRows(WhereClause where, string input, DbConn
                     {
                         Logger.Write(TraceEventType.Error, ex.ToString());
                     }
+                    finally{
```

Comment:
Please see my comments about where to add the try-finally block, if you add it here and ExecuteReader fails, the parameter collection won't be cleared.